packages/genesys-spark-components/src/components/beta/gux-avatar/example.html

```diff
@@ -769,7 +769,7 @@ <h2>Change Photo (large only)</h2>
 
   .size-default {
     padding-left: 8px;
-    border-inline-start: 1px solid var(--gse-core-color-cornflower-600);
+    border-inline-start: 1px solid #626e84;
```

Comment:
All of these hardcoded values will get picked up in a future step when we review for possible missing symantec and ui tokens